libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1396,30 +1396,22 @@ public void onMediaUploadProgress(final String localMediaId, final float progres
     }
 
     @Override
-    public void onMediaUploadFailed(final String localMediaId, final EditorFragmentAbstract.MediaType
-            mediaType, final String errorMessage) {
+    public void onMediaUploadFailed(final String localMediaId) {
```

Comment:
I've removed two parameters here
1. ErrorMessage - this parameter wasn't being used at all
2. MediaType - this parameter was used in the switch. But if you check the switch, it executes the same code no matter the mediaType. The only case when the mediaType has some effect on the code path is when it's null. However, [this](https://github.com/wordpress-mobile/WordPress-Android/blob/ee93098546f355636c6151b39c61ccd3de6a8836/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragmentAbstract.java#L165) method used to get the MediaType never returns null. Therefore, I believe it's safe to remove it altogether.